std/test/__init__.py

```diff
@@ -0,0 +1,619 @@
+import io
+import datetime
+import pytimeparse
+from unittest import TestCase
+from blspy import AugSchemeMPL, G1Element, PrivateKey
+
+from clvm.serialize import sexp_from_stream
+from clvm import SExp
+
+from lib.std.types.sized_bytes import bytes32
+from lib.std.types.ints import uint64
+from lib.std.util.keys import public_key_for_index, private_key_for_index
+from lib.std.sim.node import Node
+from lib.std.types.program import Program
+from lib.std.util.condition_tools import ConditionOpcode, conditions_by_opcode
+from lib.std.spends.p2_delegated_puzzle_or_hidden_puzzle import puzzle_for_pk, solution_for_delegated_puzzle, calculate_synthetic_secret_key # standard_transaction
+from lib.std.spends.defaults import DEFAULT_HIDDEN_PUZZLE, DEFAULT_HIDDEN_PUZZLE_HASH
+from lib.std.sim.default_constants import DEFAULT_CONSTANTS
+from lib.std.types.spend_bundle import SpendBundle
+from lib.std.types.coin_solution import CoinSolution
+from lib.std.types.coin_signature import sign_coin_solutions
+from lib.std.types.coin import Coin
+from lib.std.spends.chialisp import sha256tree
+from lib.std.types.program import Program
+from lib.std.util.hash import std_hash
+
+CONDITION_CREATE_COIN = 51
```

Comment:
i didn't take them from there because they're bytes objects, but I suppose it wouldn't matter for chialisp output.
Still, I'd imagine that factoring out the integral values (easier to understand and work with in python) from the bytes objects would make sense.  I'll pr that separately.

Reply:
Yeah I think you should import these, it's consistent with the rest of the source code and you're hopefully never manipulating the conditions, they are just constants so it shouldn't matter what's underneath as long as it works.

Reply:
i wasn't really thinking 'these things just get turned into atoms even if they're normally presented as numbers', so switched